subsystem/ipv6_test.go

```diff
@@ -0,0 +1,300 @@
+package subsystem
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/alecthomas/units"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/client/installer"
+	"github.com/openshift/assisted-service/models"
+)
+
+var (
+	validHwInfoV6 = &models.Inventory{
+		CPU:    &models.CPU{Count: 16},
+		Memory: &models.Memory{PhysicalBytes: int64(32 * units.GiB)},
+		Disks: []*models.Disk{
+			{DriveType: "SSD", Name: "loop0", SizeBytes: validDiskSize},
+			{DriveType: "HDD", Name: "sdb", SizeBytes: validDiskSize}},
+		Interfaces: []*models.Interface{
+			{
+				IPV6Addresses: []string{
+					"1001:db8::10/120",
+				},
+			},
+		},
+		SystemVendor: &models.SystemVendor{Manufacturer: "manu", ProductName: "prod", SerialNumber: "3534"},
+		Timestamp:    1601853088,
+	}
+)
+
+var _ = Describe("IPv6 installation", func() {
```

Comment:
Did you consider parameterizing the IPv4 installation tests so that the same scenarios can run twice - with IPv4 and IPv6?

Reply:
Yes, then we will double the duration of the tests

Reply:
I didn't mean all of them, just a subset that is likely to depend on IP addresses.

Reply:
Almost all